index.html

```diff
@@ -53,211 +53,92 @@
     <title>NEW DEVS ON THE BLOCK</title>
   </head>
   <body>
-    <header>
+    <header id="top">
+      <nav>
+        <a href="#about">About</a>
+        <a href="#schedule">Schedule</a>
+        <a href="#workshops">Workshops</a>
+        <a href="#coaches">Coaches</a>
+      </nav>
       <img
         src="images/logo.png"
         class="logo"
         alt="kickass new devs on the block logo"
       />
     </header>
     <main>
-      <section>
-        <h2>Become a NEW DEV on the Block</h2>
+      <article id="about">
+        <h2>About us</h2>
         <p>
-          Have you always wanted to build your own website but something held
-          you back? Maybe because you thought you were not that techie after
-          all? Then these two days are exactly for you! We take the first few
-          steps down the web development road together and the best thing is
-          that it’s all completely free of charge.
+          NEW DEVS ON THE BLOCK is a two-day workshop for front-end development
+          driven and supported by the fine people of the web community.
         </p>
-      </section>
-      <section>
-        <h2>Schedule</h2>
-        <h3>Day 1 - Build your own website</h3>
         <p>
-          On day 1 (8:45-5) of your first dev adventure you’ll learn the basics
-          of HTML and CSS and get the chance to build your very own website.
-          With hands-on support from your badass coaches Sara, Klara, Carolyn,
-          Martin and Nico and an awesome crew of like-minded people.
+          We will help you learn the fundamental building blocks of websites and
+          guide you through coding your first website yourself.
         </p>
-        <h3>Day 2 - Talks</h3>
+      </article>
+
+      <article id="schedule">
+        <h2>Schedule</h2>
         <p>
-          On day 2 (9-5) it’s all about sharing insights about web development,
-          finding the right resources to dive a little deeper and what can help
-          if a career within the digital ecosystem doesn’t sound like too bad of
-          an idea after all. Last but certainly not least, we want to put you in
-          the center of these two days. So please bring all your questions and
-          doubts and ask it away!
+          The workshop is divided into different topics that build on top of each other. Each topic contains a theoretical part, where we go over some slides, and a coding part, where you will turn theory into code.
         </p>
-      </section>
-      <section>
-        <h2>Talks</h2>
+
+        <h3>Topics</h3>
+        <p>Building a website requires HTML, CSS, and often JavaScript. We will go over each technology and cover relevant topics.</p>
```

Comment:
The word "requires" is pretty strong in this case. Maybe we can instead say something like "We will be building our websites with..." - Wdyt?